modules/installation-special-config-storage.adoc

```diff
@@ -18,25 +35,46 @@ This feature:
+=== Configuring an encryption threshold
+
+In {product-title}, you can specify a requirement for more than one Tang server. You can also configure the TPM v2 and Tang encryption modes simultaneously, so that the boot disk data can be decrypted only if the TPM secure cryptoprocessor is present and the Tang servers can be accessed over a secure network. 
 
-* TPM v2: This is the preferred mode. TPM v2 stores passphrases in a secure cryptoprocessor.
-* Tang: To use Tang to encrypt your cluster, you need to use a Tang server. Clevis implements decryption on the client side.
+You can use the `threshold:` attribute in your Butane configuration to define the minimum number of TPM v2 and Tang encryption conditions that must be met for decryption to occur. The threshold is met when the stated value is reached through any combination of the declared conditions. For example, the `threshold:` value of `2` in the following configuration can be reached by accessing the two Tang servers, or by accessing the TPM secure cryptoprocessor and one of the Tang servers:
+
+.Example Butane configuration
+
+[source,yaml]
+----
+boot_device:
+  layout: x86_64 
+  luks: 
+    tpm2: true <1>
+    tang: <2>
+      - url: http://tang1.example.com:7500
+        thumbprint: jwGN5tRFK-kF6pIX89ssF3khxxX
+      - url: http://tang2.example.com:7500
+        thumbprint: VCJsvZFjBSIHSldw78rOrq7h2ZF
+    threshold: 2 <3>
+----
+<1> Include this field if you want to use a Trusted Platform Module (TPM) to encrypt the root file system.
+<2> Include this section if you want to use one or more Tang servers.
+<3> Specify the minimum number of TPM v2 and Tang encryption conditions that must be met for decryption to occur. The default value is `1`. See the _Configuring an encryption threshold_ section for more information on this topic.
 
 [IMPORTANT]
 ====
-The use of Tang encryption mode to encrypt your disks is only supported for bare metal and vSphere installations on user-provisioned infrastructure.
+The default `threshold:` value is `1`. If you include multiple encryption conditions in your configuration but do not specify a threshold, decryption can occur if any of the conditions are met.
 ====
 
 [NOTE]
 ====
-On previous versions of {op-system}, disk encryption was configured by specifying `/etc/clevis.json` in the Ignition config. That file is not supported in clusters created with {product-title} 4.7 or above, and disk encryption should be configured using the following procedure.
+If you require both TPM v2 and Tang for decryption, the value of the `threshold:` attribute must equal the total number of stated Tang servers plus one. If the `threshold:` value is lower, it is possible for the threshold to be reached by using one of the encryption modes only. For example, if `tmp2:` is set to `true` and you specify two Tang servers, a threshold of `2` can be met by accessing the two Tang servers even if the TPM secure cryptoprocessor is not available.
```

Comment:
Colons, and also s/tmp2/tpm2/.

Reply:
I have resolved both suggestions. Good catch!

Reply:
Still needs s/tpm2:/tpm2/.

Reply:
I posted the comment after updating my local branch, briefly before I pushed my changes to the PR. I do this because once you push the changes, the comments disappear from the "Files changed" view. I will consider changing my workflow.

Reply:
No worries, I was looking at the interdiff after you force-pushed.  Check again, though; you still have a trailing colon.

Reply:
Sorry, for some reason I thought your second comment referenced the s/tmp2/tpm2/ suggestion again. Also, I was thinking about the colons reference specifically in context to `threshold:`. I have now applied s/tpm2:/tpm2/. Thanks!